packages/teams-js/src/public/secondaryBrowser.ts

```diff
@@ -13,7 +13,8 @@ import { runtime } from './runtime';
  */
 export namespace secondaryBrowser {
   /**
-   * Open a URL in the secondary browser aka in-app browser
+   * Open a URL in the secondary browser, aka in-app browser. 
```

Comment:
Let's avoid abbreviations like "aka". How about:

Opens a URL in the secondary browser (also known as in-app browser).